### DISCUSSION TITLE
ublock javascript not working

### DISCUSSION THREAD

**Discussion by j3lf** [3kh0/ext-remover#780] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I copied and pasted everything it said to and added the filter to

**Comment by RavenChill-Dev** [comment, marked as answer]
Could you be more specific? I do not understand. If your saying the JS Bookmarklets don't work, its most likely your school have blocked bookmarklets via admin console. Sorry bud, try LTMEAT Instead.
